src/Forms/Shared/Samples/GraphicsOverlay/AddGraphicsRenderer/AddGraphicsRenderer.xaml.cs

```diff
@@ -179,5 +199,25 @@ private Geometry MakeHeartGeometry(MapPoint center, double sideLength)
             builder.AddPart(newPart);
             return builder.ToGeometry();
         }
+
+        private Polygon MakeEllipseGeometry()
+        {
+            // create parameters and set all the parameters
+            GeodesicEllipseParameters parameters = new GeodesicEllipseParameters();
+            parameters.Center = new MapPoint(70e5, 10e5, SpatialReferences.WebMercator);
+            parameters.GeometryType = GeometryType.Polygon;
+            parameters.SemiAxis1Length = 1000;
+            parameters.SemiAxis2Length = 2000;
+            parameters.AxisDirection = 45;
+            parameters.MaxPointCount = 100;
```

Comment:
There's a good opportunity to use an object initializer

```suggestion
            GeodesicEllipseParameters parameters = new GeodesicEllipseParameters
            {
                Center = new MapPoint(70e5, 10e5, SpatialReferences.WebMercator),
                GeometryType = GeometryType.Polygon,
                SemiAxis1Length = 1000,
                SemiAxis2Length = 2000,
                AxisDirection = 45,
                MaxPointCount = 100,
                AngularUnit = AngularUnits.Degrees,
                LinearUnit = LinearUnits.Kilometers,
                MaxSegmentLength = 20
            };
```